### DISCUSSION TITLE
How to use GETDATE() SQL server function in Prisma?

### DISCUSSION THREAD

**Discussion by max-programming** [prisma/prisma#12705] [Q&A]
In SQL Server, there's a function in the query called `GETDATE()` which fills the current date. 

How can I implement that in Prisma to insert the current date in a row while creating or updating it?

I used `@default(now())` but that didn't work and it was null when I queried it. Can it be possible because I had set that column to be optional?

```prisma
model User {
  createdAt DateTime? @db.DateTime @default(now())
}
```

**Comment by janpio** [comment]
I don't think we have support for `GETDATE()` yet. You should open a feature request. Thanks.

**Reply by max-programming** [reply]
So even if I set the type to be optional, and a default value, it must fill the default value right? If I don't pass it in the `create()` method.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
